src/app/modules/item/pages/group-progress-grid/group-progress-grid.component.html

```diff
@@ -20,6 +20,38 @@
     <p class="empty-message" *ngSwitchCase="'Groups'" i18n>This group has no sub-groups</p>
   </ng-container>
 
+  <p-overlayPanel #progressOverlayPanel appendTo="body" styleClass="user-progress-overlay-panel">
```

Comment:
I have not reviewed this last commit yet. But it is not straightforward, shouldn't this be moved to a separated component? More things will be added to it in the future, making it even more complex.

Reply:
It's the same as before, only it's merged into one overlay for all rows instead of one by row.
I can move to it to a new component, although I think it's not really in the scope of this PR.

Reply:
Fixed